app/shared/side-navbar/SideNavbar.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react';
+import FontAwesome from 'react-fontawesome';
+
+import Footer from 'shared/footer';
+import Navbar from 'shared/navbar';
+import Sidebar from 'shared/sidebar';
+
+export class SideNavbar extends Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  state = { open: false };
+
+  onToggleSideBar = () => {
+    this.setState({ open: !this.state.open });
+  }
+
+  onSetSidebarOpen = (open) => {
+    this.setState({ open });
+  }
+
+  renderSidebarContent() {
+    return (
+      <div className="app-SideNavbar__content">
+        <Navbar />
+        <Footer />
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <Sidebar
+        onSetOpen={this.onSetSidebarOpen}
+        open={this.state.open}
+        rootClassName="app-SideNavbar"
+        sidebar={this.renderSidebarContent()}
+      >
```

Comment:
If docked prop is passed, it won't show the black overlay on top of the main content and the main component would be clickable without closing the navbar.